resources/assets/components/CampaignIdSingle/index.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { map, isEmpty } from 'lodash';
+
+import './campaignidsingle.scss';
+
+import Action from '../Action';
+import UploaderModal from '../UploaderModal';
+import ModalContainer from '../ModalContainer';
+// import CreateActionModal from '../CreateActionModal';
+
+class CampaignIdSingle extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      displayCreateActionModal: false,
+    };
+
+    // this.showCreateActionModal = this.showCreateActionModal.bind(this);
+    // this.hideCreateActionModal = this.hideCreateActionModal.bind(this);
+  }
+
+  // Open the create action modal
+  // showCreateActionModal(campaign, event) {
+  //   event.preventDefault();
+
+  //   this.setState({
+  //     displayCreateActionModal: true,
+  //     campaign,
+  //   });
+  // }
+
+  // Close the create action modal
+  // hideCreateActionModal(event) {
+  //   if (event) {
+  //     event.preventDefault();
+  //   }
+
+  //   this.setState({
+  //     displayCreateActionModal: false,
+  //     campaign: null,
+  //   });
+  // }
+
```

Comment:
same with the above - ok to leave commented out to come back to when there is time to try to add the modal again?

Reply:
I think that is fine for now!